.github/actions/spelling/allow.txt

```diff
@@ -264,6 +264,7 @@ IPortable
 ipmo
 ISAPPROVEDFOROUTPUT
 isspace
+istavailable
```

Comment:
```suggestion
listavailable
```
nit: allow the full violating string here?

Reply:
Let's see if the spelling bot cooperates :)

Reply:
Nope. reverting

Reply:
This GitHub action appears to support patterns to ignore, so maybe instead of allowing these partial words, have it ignore anything within backticks?